src/processor/operator/ddl/alter.cpp

```diff
@@ -7,6 +7,31 @@
 namespace kuzu {
 namespace processor {
 
+std::string AlterPrintInfo::toString() const {
+    std::string result = "Operation: ";
+    switch (alterType) {
+    case common::AlterType::RENAME_TABLE:
+        result += "Rename Table";
```

Comment:
We probably want to be more specific here. Rename table X to Y, Add property Z, ...